### DISCUSSION TITLE
Dynamic fields in the UI

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#3150] [Feature Requests]
Hi.

Sometimes I need some field to behave dynamically. Like if one field is present. do not show the other. This is just one example. Do you have plans for smth like this?

**Comment by benhaynes** [comment]
Yes, I've added a ticket for "Conditional Fields", which is something we've been thinking about how to properly add for a while now. Ref: https://github.com/directus/directus/issues/3151

**Comment by rijkvanzanten** [comment]
This has since been added as Conditional Fields!
